### ISSUE TITLE
what the hell? ransomware alter?

### ISSUE THREAD

**Issue by soulspirits** [randy3k/radian#293]
![Snipaste_2021-07-06_12-22-30](https://user-images.githubusercontent.com/1639283/124542269-a95d9100-de55-11eb-9820-a351f65a72d5.png)
one of my antivirus software pops up ransomware alter when I try to execute the following code within vscode 

`install.packages('rmarkdown')
install.packages('reticulate')
install.packages("Rcpp")
library(rmarkdown)`

any suggestion?

**Comment by randy3k** [comment]
It's interesting. Did you install the packages from CRAN?

**Comment by soulspirits** [comment]
yes,use


pip3 install -U radian




| |
Stephen
|
|
***@***.***
|
签名由网易邮箱大师定制
On 7/6/2021 13:40，Randy ***@***.***> wrote：

It's interesting. Did you install the packages from CRAN?

**Comment by randy3k** [comment]
I meant the R packages.

**Comment by soulspirits** [comment]
yep,also use official CRAN and install.packages() command within radian console.


| |
Stephen
|
|
***@***.***
|
签名由网易邮箱大师定制
On 7/6/2021 14:02，Randy ***@***.***> wrote：

I meant the R packages.

**Comment by randy3k** [comment]
TBH, I have no idea why the pdf was detected as a ransomeware. What happen if you scan that pdf directly?

**Comment by soulspirits** [comment]
Maybe false alarm from "360 internet security", no other ( aslo installed symantec on this computer) anti-virus detected ransomeware behavious. 


Anyway,thanks for the replies. 


| |
Stephen
|
|
***@***.***
|
签名由网易邮箱大师定制
On 7/6/2021 14:24，Randy ***@***.***> wrote：

TBH, I have no idea why the pdf was detected as a ransomeware. What happen if you scan that pdf directly?
